doctest/extensions/mpi_reporter.h

```diff
@@ -66,6 +66,7 @@ struct MpiConsoleReporter : public ConsoleReporter {
       return nullStream;
     }
   }
+  std::vector<std::pair<std::string, int>> m_failure_str_queue = {};
```

Comment:
```suggestion
  std::vector<std::pair<std::string, int>> m_failure_str_queue;
```

I think this is not needed (?)

Reply:
True, but I think it is more consistent with the member declarations seen in other classes in the file.